test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,67 @@
+package seedu.addressbook.common;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import java.util.ArrayList;
+
+public class UtilsTest {
+    private Object[] list_noNull, list_oneNull, list_someNull;
+    private ArrayList<Object> arrayList_noDuplicate = new ArrayList<Object>();
+    private ArrayList<Object> arrayList_oneDuplicate = new ArrayList<Object>();
+    private ArrayList<Object> arrayList_someDuplicates = new ArrayList<Object>();
+    
+    @Before
+    public void setupLists() {
+        this.list_noNull = new Object[]{"hello", "world"};
```

Comment:
hmm. you can populate the lists within each particular test case actually. Setup is usually for things that need to be done each time before a test case.